motion/motionprocessor.go

```diff
@@ -109,6 +112,11 @@ func (mp *MotionProcessor) Process(rawFrame []byte) error {
 		mp.stopConstantRecorder()
 		return err
 	}
+	if mp.CurrentFrame == math.MaxUint32 {
+		mp.CurrentFrame = 0
+	} else {
+		mp.CurrentFrame += 1
+	}
```

Comment:
Is this needed?
```
var a uint32 = math.MaxUint32
log.Println(a)
a = a + 1
log.Println(a)
```
will return 
```
4294967295
0
```
Also that will take a very long time to get to that frame